todo-app/src/Components/todoItem.js

```diff
@@ -0,0 +1,21 @@
+//jshint esnext: true
+
+import React from 'react';
+import './styles/todoItem.css';
+
+
+export default class TodoItem extends React.Component {
+    render (){
+        const {description, deadline, done, id } = this.props.data;
+        return (
+            <li className="todoItem">
+                <div>
+                    <span>{deadline}</span>
+                    <label for={`item${id}`} className={done ? 'markDone' : 'markUndone'}></label>
```

Comment:
If you open your browser console after running your app, you'll see this error:

    Warning: Invalid DOM property `for`. Did you mean `htmlFor`?

Check out the [React docs here](https://reactjs.org/docs/dom-elements.html#htmlfor). Because `for` is a reserved word, you have to use `htmlFor` instead with JSX.